src/test/java/com/cdancy/bitbucket/rest/features/HookApiMockTest.java

```diff
@@ -46,7 +46,7 @@
     private final String deleteMethod = "DELETE";
     private final String putMethod = "PUT";
 
-    private final String restApiPath = "/rest/api/";    
+    private final String restApiPath = "/rest/api/";
```

Comment:
:warning: This final field could be made static (#QR176:49#)-PMD_FinalFieldCouldBeStatic